src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -0,0 +1,38 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class WinningLotto {
+    private final List<LottoNumber> lottoNumbers;
```

Comment:
```suggestion
    private final Lotto lotto;
```

위와 같이 Lotto 를 갖고 있는게 더 어울려보여요!
로또가 6개에서 7개로 변한다거나, 1-50 까지의 숫자 요구사항이 변한다면, 같이 변해야하기 때문이에요 🤔